duckduckgo/duckduckgo-privacy-extension
# Brad/is first party fix

**Reviewer:** @sammacbeth @dharb 



## Description:

This PR switches the 3rd party check from just tldts which only checked the hostname to using entity data from TR. This should fix any issues of first party cookies being blocked where example.com is setting cookies from otherexample.com.

## Steps to test this PR:
1. Verify tests work at https://privacy-test-pages.glitch.me/privacy-protections/storage-blocking
2. See asana task for other notes

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
